tests/integration/test_transcribe.py

```diff
@@ -1,7 +1,127 @@
+
+        job = transcribe_client.get_transcription_job(TranscriptionJobName=job_name)
+        snapshot.match("TranscriptionJob", job)
+
+        # delete the job again
+        transcribe_client.delete_transcription_job(TranscriptionJobName=job_name)
+
+        # check if job is gone
+        with pytest.raises((ClientError, NotFoundException)) as e_info:
+            transcribe_client.get_transcription_job(TranscriptionJobName=job_name)
+
+        snapshot.match("GetError", e_info.value.response)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(
+        paths=["$..TranscriptionJob..Settings", "$..TranscriptionJob..Transcript", "$..Error..Code"]
+    )
+    def test_get_transcription_job(self, transcribe_client, transcribe_create_job, snapshot):
+        job_name = transcribe_create_job(test_file=os.path.abspath("files/en-gb.wav"))
+
+        job = transcribe_client.get_transcription_job(TranscriptionJobName=job_name)
+
+        snapshot.match("GetJob", job)
+
+        with pytest.raises((ClientError, NotFoundException)) as e_info:
+            transcribe_client.get_transcription_job(TranscriptionJobName="non-existent")
+
+        snapshot.match("GetError", e_info.value.response)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(
+        paths=["$..NextToken", "$..TranscriptionJobSummaries..OutputLocationType"]
+    )
+    def test_list_transcription_jobs(self, transcribe_client, transcribe_create_job, snapshot):
+        transcribe_create_job(test_file=os.path.abspath("files/en-gb.wav"))
+
+        jobs = transcribe_client.list_transcription_jobs()
+
+        # there are potentially multiple transcription jobs on AWS - ordered by creation date
+        # we only care about the newest one that we just created
+        jobs["TranscriptionJobSummaries"] = jobs["TranscriptionJobSummaries"][0]
+
+        snapshot.match("ListJobs", jobs)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(paths=["$..Error..Code"])
+    def test_failing_deletion(self, transcribe_client, snapshot):
+        # successful deletion is tested in the happy path test
+        # this tests a failed deletion
+        with pytest.raises((ClientError, NotFoundException)) as e_info:
+            transcribe_client.delete_transcription_job(TranscriptionJobName="non-existent")
+
+        snapshot.match("MissingLanguageCode", e_info.value.response)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(
+        paths=["$..MissingLanguageCode..Message", "$..MalformedLanguageCode..Message"]
+    )
+    def test_failing_start_transcription_job(
+        self, s3_client, s3_bucket, transcribe_client, snapshot
+    ):
+        transcription_job = f"test-transcribe-{short_uid()}"
+        test_key = "test-clip.wav"
+        file_path = os.path.abspath("files/en-gb.wav")
+        with open(file_path, "rb") as f:
+            s3_client.upload_fileobj(f, s3_bucket, test_key)
+
```

Comment:
Worth considering making a function out of this since this is also used elsewhere

Reply:
The only place I see this is in `test_transcribe.py` and `fixtures.py` (but in fixtures it's a bit more complicated). Abstracting this would hurt readability IMHO - but happy to discuss